drivers/net/can/spi/mcp25xxfd/mcp25xxfd-core.c

```diff
@@ -2736,6 +2874,31 @@ static int mcp25xxfd_probe(struct spi_device *spi)
 	else if (IS_ERR(reg_xceiver))
 		return PTR_ERR(reg_xceiver);
 
+	err = device_property_read_u32(&spi->dev, "mcp25xxfd,clkout-divisor", &clkodiv);
```

Comment:
Please implement the `.set_rate` callback: https://elixir.bootlin.com/linux/v5.7/source/include/linux/clk-provider.h#L150 and remove the DT property.

Reply:
If you only need a divider of 1, you can hardcode this and add `.set_rate` in a later patch. But setting the divider from DT is no option.